src/main/java/org/opensearch/security/ssl/DefaultSecurityKeyStore.java

```diff
@@ -30,15 +30,7 @@
 import java.security.PrivilegedExceptionAction;
 import java.security.cert.CertificateParsingException;
 import java.security.cert.X509Certificate;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Could you please, run `gradle spotlessCheck spotlessApply`?

Reply:
Looks like that didn't remove the wildcard imports. Can you adjust your IDE settings to prevent wildcard imports?

Reply:
True `spotlessCheck spotlessApply` does not do it.

Reply:
Done